subprojects/testing-jvm/testing-jvm.gradle

```diff
@@ -24,6 +24,8 @@ dependencies {
 
     compile libraries.junit
     compile libraries.testng
+    compile libraries.kryo
```

Comment:
Looks like `kryo` is removed in https://github.com/gradle/gradle/commit/4f2f2b6822bd90774830729452e3f8536ff1b0ff#diff-b3b7270db0152b60fc02dd4db1a53d4a , why?

Reply:
It was probably leaking. Why did you have to re-introduce?

Reply:
No, it's just a merging issue, I've removed it.